services/newsfeed/shared/drupal.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {};
```

Comment:
Please consider renaming the `shared` directory to `sources` instead.

Reply:
I was planning on keeping the same dir structure (jobs & methods use shared files). Is there any reason to change it? Esp. when there are only 3-5 files?

Reply:
Then to keep the directory structure consistent with the other microservices should we not move these files to `shared/newsfeed`?
The files maintained directly under the `shared` directory (within `core` microservice) are highly generic files and not directly related to the API endpoints.

Reply:
The `core` microservice needs directories for two reasons: 1) the number of files is already high 2) some files got the same names and might require long prefixes otherwise. None of them is the case here. When the scope becomes bigger, the files will be organised into directories.